docker/blackduck-alert-db/alertdb-pg-migrate.sh

```diff
@@ -86,6 +77,26 @@ function _validate_migration_viability() {
   _logEnd
 }
 
+function _create_openshift_custom_conf() {
+  _logStart
+  local openshiftConfFile="/var/lib/pgsql/openshift-custom-postgresql.conf"
+  local openshiftTemplateFile="${openshiftConfFile}.template"
+
+  if [ ! -s "${openshiftConfFile}" ]; then
+    export POSTGRESQL_MAX_CONNECTIONS=${POSTGRESQL_MAX_CONNECTIONS:-100}
+    export POSTGRESQL_MAX_PREPARED_TRANSACTIONS=${POSTGRESQL_MAX_PREPARED_TRANSACTIONS:-0}
+    export POSTGRESQL_SHARED_BUFFERS=${POSTGRESQL_SHARED_BUFFERS:-32MB}
+    export POSTGRESQL_EFFECTIVE_CACHE_SIZE=${POSTGRESQL_EFFECTIVE_CACHE_SIZE:-128MB}
+
+    envsubst < "${openshiftTemplateFile}" > "${openshiftConfFile}"
+    _checkStatus $? "Creating openshift custom conf file"
+  else
+    _logIt "Custom conf file already exists"
+  fi
+
+  _logEnd
+}
+
 function _update_existing_data_ownership() {
```

Comment:
Create conf file needed for Centos upgrades. envsubst does a search and replace on environment variables. This is for issue 2